config/default.json.dist

```diff
@@ -6,5 +6,10 @@
       "baseUrl": "your URL",
       "route": "'your route'",
       "listener": "your listener"
+      "filterby": "your filter",
```

Comment:
`testTitleFilter`? 🤷‍♂ (a page speed item in statuscake is a test https://www.statuscake.com/api/Page%20Speed/List%20Pagespeed%20Test.md)

Reply:
☝️